### PR TITLE
Register Janco.is-a.dev

### PR DESCRIPTION
**Pull request by JancoNel**
Register Janco.is-a.dev with CNAME record pointing to https://janconel.github.io/Personalwebsite/.

### CODE REVIEW

File: domains/janco.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "JancoNel",
+           "email": "",
+           "discord": "1264555376918134796"
+        },
+    
+        "record": {
+            "CNAME": "https://janconel.github.io/personalwebsite/"
```
Comment (orxngc) [inline domains/janco.json:9]:
```suggestion
            "CNAME": "https://janconel.github.io"
```
CNAME records cannot contain slashes (`/`).

**Comment by orxngc** [comment]
And your site is incomplete. Please add some content.

**Comment by JancoNel** [comment]
Hi 

- I made the changes to record , was unsure about the what I should put in it becuae the last couple of times I also messed it up.

- About the incomplete website , I thought I would sort the domain/subdomain out before I start focusing on that

- Does the fact that it is incomplete cause any problems? @orxngc

File: domains/janco.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "JancoNel",
+           "email": "",
+           "discord": "1264555376918134796"
+        },
+    
+        "record": {
+            "CNAME": "https://janconel.github.io"
```
Comment (CuteDog5695) [inline domains/janco.json:9]:
```suggestion
            "CNAME": "janconel.github.io"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
Also about your website, at the moment its just a simple "Hello World!" with just some plain text. As from the comment you send, I understand that you first wanted the domain and then work on the websites, but we usually don't merge PRs which aren't "complete". If you want for this PR to be merged, you can add like a about me and then I'll merge the PR.

**Comment by JancoNel** [comment]
Alright , I made the chanhges to janco.json , going to make some improvements to the site to make it look more complete now

**Comment by JancoNel** [comment]
Done , the website looks a lot more complete now after a bit of work on it 

https://janconel.github.io/Personalwebsite/

**Comment by JancoNel** [comment]
Screw it yall are taking your time im oppening a new PR

**Pull request closed without merging**